readme.md

```diff
@@ -46,8 +46,26 @@ To use a custom SVG, provide its url in the `src` attribute to request the exter
 <ion-icon src="/path/to/external/file.svg"></ion-icon>
 ```
 
+### Custom Asset Path
```

Comment:
Would it makes sense to have this be `####` to show up under "Custom Icons" above? They seem related.

Reply:
Added another `#`